static/src/javascripts/projects/common/modules/ui/subscription-banner.js

```diff
@@ -55,6 +56,29 @@ const hideBannerInTheseRegions: ReaderRevenueRegion[] = [
 const subscriptionUrl = `${subscriptionHostname}/subscribe/digital?INTCMP=gdnwb_copts_banner_subscribe_SubscriptionBanner&acquisitionData=%7B%22source%22%3A%22GUARDIAN_WEB%22%2C%22campaignCode%22%3A%22subscriptions_banner%22%2C%22componentType%22%3A%22${COMPONENT_TYPE}%22%2C%22componentId%22%3A%22${OPHAN_EVENT_ID}%22%7D`;
 const signInUrl = `${signinHostname}/signin?utm_source=gdnwb&utm_medium=banner&utm_campaign=SubsBanner_Exisiting&CMP_TU=mrtn&CMP_BUNIT=subs`;
 
+const hasAcknowledged = bannerRedeploymentDate => {
+    const redeploymentDate = new Date(bannerRedeploymentDate).getTime(); // 2 Dec 2019 @ 5:00
```

Comment:
Question: when we release - there will already be an object in S3 with this previous deploy date?

Reply:
I've added deployment dates for each region to AWS S3 buckets